src/devices/Blinkt/Blinkt.cs

```diff
@@ -0,0 +1,194 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.Gpio;
+using System.Threading;
+using System.Drawing;
+
+namespace Iot.Device.Blinkt
+{
+    /// <summary>
+    /// Driver for the Pimoroni Blinkt LED strip for Raspbery Pi
+    /// https://shop.pimoroni.com/products/blinkt
+    ///
+    /// This is a strip of 8 pixels that can be indendently controlled over GPIO.
+    ///
+    /// Setting the color is a 2-step process:
+    /// Set the color using Clear, SetPixel, or SetAll
+    /// Call Show to update the physical pixels
+    /// </summary>
+    public class Blinkt : IDisposable
+    {
+        private const int DAT = 23;
+        private const int CLK = 24;
```

Comment:
The pin numbers should be provided as constructor arguments. One might want to attach the strip to other pins as well. You could also use `GpioPin`-type arguments instead of numbers.